src/test/groovy/org/prebid/server/functional/tests/privacy/GdprAmpSpec.groovy

```diff
@@ -369,4 +370,64 @@ class GdprAmpSpec extends PrivacyBaseSpec {
         assert response.ext?.warnings[PREBID]*.message ==
                 ["Parsing consent string: ${tcfConsent} failed. TCF policy version ${invalidTcfPolicyVersion} is not supported" as String]
     }
+
+    def "PBS amp should try to fetch vendor list by exponential backoff and emit error when vendor list response is absent"() {
```

Comment:
Not clear from naming what it should doing:
`If we already tried to fetch it and not enough time have passed since last try - return an error.`
maybe `PBS amp should emit the same error without a second GVL list request if a retry is too soon for the exponential-backoff`

Reply:
and add the test where is will retry after `exponential-backoff` time

Reply:
Discussed